utxonursery.go

```diff
@@ -1069,7 +1069,7 @@ type contractMaturityReport struct {
 	limboBalance btcutil.Amount
 
 	// recoveredBalance is the total value that has been successfully swept
-	// back to the user's wallet.
+	// back to the user's Wallet.
```

Comment:
IMO none of the Wallet capitalization comments in this commit need to be touched. this commit should only touch the changes that affect compilation